### PR TITLE
docs(issues): update template

### PR DESCRIPTION
**Pull request by gotbadger**
## Description
Updates based on whats normally reported on bug reports and whats most important when im triaging them.

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

**Comment by gotbadger** [comment]
@markmichon let me know what you think about this.

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -6,9 +6,14 @@ about: Report a bug or unexpected behavior.
 
 <!--- Provide a general summary of the issue in the Title above -->
 
-## Description
+## Description & Reporduction
```
Comment (cfabianski) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md]:
```suggestion
## Description & Reproduction
```

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 2]
```diff
@@ -22,24 +27,14 @@ about: Report a bug or unexpected behavior.
...
-
-<!--- How has this bug affected you? What were you trying to accomplish? -->
-
 ## Your Environment
 
 <!--- Include as many relevant details about the environment you experienced the bug in -->
 
-- Version used:
-- Target codebase stack (e.g. Ruby):
 - Operating System and version:
```
Comment (cfabianski) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:34]: Maybe we should output the OS and such as part of the version (or maybe have a command to display this kind of info).

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 3]
```diff
@@ -22,24 +27,14 @@ about: Report a bug or unexpected behavior.
...
-- Link to your project or code sample:
```
Comment (cfabianski) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:45]: Kinda like this one 🤷

Comment (gotbadger) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md:45]: It's included in the description message now. I found when I was putting in reports i would be talking about it in the description then have to enter it twice.

**Review by markmichon — APPROVED** [review]

**Pull request merged**